distributions/debian/control

```diff
@@ -1,43 +1,43 @@
 Source: jamulus
 Section: sound
 Priority: optional
-Maintainer: "Marc Landolt jr" <debian@marclandolt.ch>
+Maintainer: "The Jamulus Development Team" <team@jamulus.io>
 Build-Depends:
  debhelper (>= 9),
  libjack-jackd2-dev,
  qtbase5-dev,
  qttools5-dev-tools,
-Standards-Version: 3.9.7
+Standards-Version: é&%JAMVERSION%&è
```

Comment:
That seems wrong to me. I know that this has been there and it looks like some magic string which someone tried to replace with the Jamulus version, but this is simply wrong. Standards-Version describes the Standard for the control file. This is a (Debian-specific number](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-standards-version).